align/config/basic_template.sp

```diff
@@ -1,104 +1,129 @@
 .subckt NMOS_S  D G S
+* @: Generator(pattern='CS', body=True)
```

Comment:
@arvuce22 
I have added generator constraints for primitives:
Can you please modify the primitive/main.py to use it properly?